code_schemes/county.json

```diff
@@ -0,0 +1,597 @@
+{
+    "SchemeID": "Scheme-af6c3ef0",
+    "Name": "County",
+    "Version": "0.0.0.1",
+    "Codes": [
+       {
+            "DisplayText": "nyeri",
```

Comment:
What's the logic for the ordering here?

Reply:
Project specific for now. Based on current Icraf target areas .